datarobot_batch_scoring/writer.py

```diff
@@ -73,7 +73,8 @@ def create(cls, resume, n_samples, out_file, pid, lid,
             is_resume = None
             if resume:
                 is_resume = True
-            if is_resume is None:
+            # Ask if resume option wasn't specified
+            if is_resume is None and resume is None:
```

Comment:
It's a bit confusing to have `is_resume` and `resume` at the same place. I think we can simplify everything if we write something like:

```python
         file_context = ContextFile(pid, lid, n_samples, keep_cols)

         # if resume wasn't specified and there's a context to resume
         if resume is None and file_context.exists():
            resume = ui.prompt_yesno('Existing run found. Resume')

         if resume:
             ctx_class = OldRunContext
         else:
             ctx_class = NewRunContext
 
         return ctx_class(n_samples, out_file, pid, lid, keep_cols, n_retry,
                          delimiter, dataset, pred_name, ui, file_context,
                          fast_mode, encoding, skip_row_id, output_delimiter)
```